### ISSUE TITLE
Life loss from an attack counts toward stun

### ISSUE THREAD

**Issue by tankbard** [tankbard/battlecon-ai#2]
Specifically, the AA: life loss from Shekhtur's Brand appears to contribute to determining whether the opponent is stunned.

**Comment by BGazazel** [comment]
I don't think this is right. Here is example where Shekhtur's Brand dmg + life loss = 6, and Adjena SG = 5 still, Adjena perform her action. May be you miss that Brand ignores stun guard at prioriy 6+ and will stun always with or without life steeling /except stun imunity of course/. Hope that i get this issue right.
![shektur_brand](https://cloud.githubusercontent.com/assets/18679815/15742295/a4ebf624-28c6-11e6-97f9-b10f69f60a6f.jpg)
